versioned_sidebars/version-2.0.0-sidebars.json

```diff
@@ -22,11 +22,9 @@
       "collapsible": true,
       "collapsed": false,
       "items": [
-        "running-keploy/cli-commands",
         "running-keploy/rename-testcases",
-        "running-keploy/docker-tls",
         "running-keploy/configuration-file",
-        "running-keploy/custom-mocks"
+        "running-keploy/cli-commands"
```

Comment:
`docker-tls` needs to be there for the user running keploy on docker and has TLS interactions